### PR TITLE
Feat: Add `uploadRatioTarget` setting to the configuration page

### PR DESCRIPTION
**Pull request by clementmoine**
Add the `uploadRatioTarget` in the configuration page according to the newly created setting on the main app [PR on `anthonyraymond/joal` #221](https://github.com/anthonyraymond/joal/pull/221)

### CODE REVIEW

**Comment by anthonyraymond** [comment]
LGTM:
Can you please replace inputProps with `inputProps={{ min:-1, step: config.uploadRatioTarget > 0 ? 0.1 : 1 }}`

File: src/pages/settings/settings.component.tsx [hunk 1]
```diff
@@ -142,6 +142,20 @@ const Settings = (props: Props) => {
                 }}
               />
             </Grid>
+            <Grid item xs={12}>
+              <TextField
+                className={classes.formInput}
+                label="Upload ratio target"
+                type="number"
+                inputProps={{ min:-1 }}
```
Comment (anthonyraymond) [inline src/pages/settings/settings.component.tsx]:
```suggestion
                inputProps={{ min:-1, step: config.uploadRatioTarget > 0 ? 0.1 : 1 }}
```

Comment (clementmoine) [reply src/pages/settings/settings.component.tsx]: That’s a good idea ! 🙂

**Comment by anthonyraymond** [comment]
Great :) thanks a lot for your PR.
I'll publish the new release tonight !

**Pull request merged**